app/controllers/api/account.php

```diff
@@ -1068,6 +1049,8 @@
     ->label('event', 'users.[userId].sessions.[sessionId].create')
     ->label('scope', 'public')
     ->label('auth.type', 'anonymous')
+    ->label('audits.resource', 'user/{response.userId}')
+    ->label('audits.userId', '{userId}')
```

Comment:
```suggestion
    ->label('audits.userId', '{user.$id}')
```

This is what I had in mind for consistency.